cmd/tailscaled/tailscaled.go

```diff
@@ -895,3 +903,13 @@ func dieOnPipeReadErrorOfFD(fd int) {
 	f.Read(make([]byte, 1))
 	os.Exit(1)
 }
+
+type healthMessageLabel struct {
+	Severity string
+}
+
+var metricHealthMessages = usermetric.NewMultiLabelMap[healthMessageLabel](
```

Comment:
Other usermetrics are defined in relevant packages, so perhaps we should move this definition to the `health` package?